homeassistant/components/hive/binary_sensor.py

```diff
@@ -47,13 +48,36 @@
     ),
 )
 
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
```

Comment:
These should be BinarySensorEntityDescriptions

Reply:
Completed